src/Data/StoresScopedComputedFieldCallbacks.php

```diff
@@ -16,7 +16,7 @@ trait StoresScopedComputedFieldCallbacks
      * @param  string|array  $scopes
      * @param  string|array  $field
      */
-    public function computed($scopes, $field, ?Closure $callback = null)
+    public function computed($scopes, $field, ?callable $callback = null)
```

Comment:
Unfortunately changing the method signature like this is a breaking change.

Would you be able to retarget this PR to master so it can go into v6 instead?